src/Storefront/Resources/views/storefront/page/account/_page.html.twig

```diff
@@ -12,7 +12,7 @@
             {% endblock %}
 
             {% block page_account_main %}
-                <div class="account-content-main col-lg-9 offset-xl-1 col-xl-9">
+                <div class="account-content-main col-lg-9 {% if feature('v6.7.0.0') %}col-xl-10{% else %}offset-xl-1 col-xl-9{% endif %}">
```

Comment:
6.7 deprecations are already removed, just do the change and it will be fine

Reply:
Okay, I adjusted it :-)